docs/user-accounts.md

```diff
@@ -0,0 +1,113 @@
+# User accounts
+
+A user represents an identity within the server. Every user is registered and has a profile for other users to find and become [friends](friends.md) with or join [groups](groups-clans.md) and [chat](realtime-chat.md).
+
+A user can own [records](storage-record-ownership.md), share public information with other users, and login via a bunch of different social providers.
+
+## Fetch self
+
+When a user has a session you can retrieve their profile. The profile is returned for yourself and so we refer to it as "self" in client code. The profile contains lots of information which includes various "linked" social providers.
+
+```csharp fct_label="Unity"
+var message = NSelfFetchMessage.Default();
+client.Send(message, (INSelf self) => {
+  var id = Encoding.UTF8.GetString(self.Id);
+  Debug.LogFormat("User has id '{0}' and handle '{1}'.", id, self.Handle);
+  var metadata = Encoding.UTF8.GetString(self.Metadata);
+  Debug.LogFormat("User has JSON metadata '{0}'.", metadata);
+}, (INError err) => {
+  Debug.LogErrorFormat("Error: code '{0}' with '{1}'.", err.Code, err.Message);
+});
+```
+
+Some information like social IDs are private but part of the profile is visible to other users.
+
+| Public field | Description |
+| ----- | ----------- |
+| AvatarUrl | A URL with a profile picture for the user (empty by default). |
+| CreatedAt | A timestamp in milliseconds for when the user was created. |
+| Fullname | The full name for the user (empty by default). |
+| Handle | A unique nickname for the user. |
+| Id | The unique identifier for the user. |
+| Lang | The preferred language settings for the user (default is "en"). |
+| LastOnlineAt | A timestamp in milliseconds when the user was last online. |
```

Comment:
@mofirouz We don't populate it right now but will do when we have the plugins system in Nakama server. I can explain after 1.0 what I have in mind for it 😄